src/test/java/uk/gov/companieshouse/extensions/api/reasons/ReasonServiceUnitTest.java

```diff
@@ -87,9 +84,9 @@ public void canGetListOfReasons() throws ServiceException {
         reason2.setId("reason2");
         extensionRequestFullEntity.addReason(reason2);
         when(requestsService.getExtensionsRequestById(REQUEST_ID)).thenReturn(Optional.of(extensionRequestFullEntity));
-        when(reasonMapper.entityToDTO(reason1))
+        lenient().when(reasonMapper.entityToDTO(reason1))
```

Comment:
this used to run without lenient before, this looks like it is masking a possible problem

Reply:
Swapped when then return for when then answer as this allows for multiple parameters.